### PR TITLE
GH-1: hotfix announcements logic

### PR DESCRIPTION
**Pull request by Midnite1212**
Fix announcement logic, no date announcement will be shown in the back (Fixed logic to handle multiple no date announcements). Announcements will be sorted from newest dates.

### CODE REVIEW

**Comment by tamarayustian** [comment]
@Baeck016 should we make the interval field also be put as default Daily? i'm thinking since you said thats the fix you made for the events carousell at home page.
it doesnt make sense i think because sometimes there is no recurring for one-time events. but i dont know what is the code issue there

**Comment by Midnite1212** [comment]
> @Baeck016 should we make the interval field also be put as default Daily? i'm thinking since you said thats the fix you made for the events carousell at home page.
> it doesnt make sense i think because sometimes there is no recurring for one-time events. but i dont know what is the code issue there

I fixed that issue alrd, now they can input the interval as none. Doesnt always have to be daily.

File: ui/src/components/home/EventsSectionCards.js [hunk 1]
```diff
@@ -76,31 +76,24 @@ const EventsSectionCard = (props) => {
   };
 
   const getStartEndDateStr = (eventData) => {
+    const eventRenderDate = DateTime.fromISO(
```
Comment (tamarayustian) [inline ui/src/components/home/EventsSectionCards.js:79]: nice i like this. its cleaner looking code

**Review by tamarayustian — APPROVED** [review]

**Comment by Midnite1212** [comment]
Closing this pr since its based off the wrong branch

**Pull request closed without merging**